### ISSUE TITLE
I desperately need the ability to insert two images into one button of the stream deck.

### ISSUE THREAD

**Issue by SeoHoon923** [shells-dw/streamdeck-totalmix#9]
Here is the image file link : https://imgdb.in/jLJh

"TotalMix FX Control : OSC : Trigger Channel Function"
If you look at this menu, you can insert two images, so when you press the button of the Streamdeck, the current state of Output Channel 1 is displayed as Mute or not, so it is easy to distinguish.

However..
"TotalMix FX Control : OSC : Trigger Global Function"
When I use the Mute Group 1 function in this menu, I can insert only one image, so Streamdeck does not show whether the current state is muted or not.
Therefore, it is very inconvenient to run Totalmix and check it.

in conclusion
"TotalMix FX Control : OSC : Trigger Global Function"
Please update this menu so that it is possible to insert two images that can display the current status.
I really need it

**Comment by SeoHoon923** [comment]
Thanks to you, I have really comfortable controlling Totalmix.
I hope this Streamdeck plugin will never go away forever.
Thank you very much.
 
-----Original Message-----

**Comment by SeoHoon923** [comment]
Thanks to you, I have really comfortable controlling Totalmix.
I hope this Streamdeck plugin will never go away forever.
Thank you very much.
